config/report_portal.yaml.example

```diff
@@ -2,7 +2,11 @@ uuid: 12345678-1234-1234-1234-123456789012
 endpoint: https://localhost:8080
 launch: example_launch_name
 project: default_personal
-tags: [tag1, tag2, tag3]
+attributes:
+  - key: build
+    value: '0.1'
+  - value: test
+#tags: [tag1, tag2, tag3] - DEPRECATED. Use attributes instead
```

Comment:
Does ReportPortal v5 still support tags?
If not, then I guess they should be removed rather than deprecated.

Reply:
Even though the 'tags' field is not explicit in the documentation (https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md), the Report Portal API accepts it. But, when you use both configuration (tags and attributes) together, the 'attributes' values are not received by the Report Portal API.
Eg:
`curl --header "Content-Type: application/json" --header "Authorization: Bearer 039eda00-b397-4a6b-bab1-b1a9a90376d1" --request POST --data '{"name":"rp_launch","description":"My first launch on RP","startTime":"1574423221000","mode":"DEFAULT","attributes":[{"key":"build","value":"0.1"},{"value":"test"}],"tags":["TAG1"]}' http://rp.com/api/v1/rp_project/launch`
In the above case, the launch only gets the 'tags' value (and ignores the 'attributes' field)

Because of it I inserted this verification:
```ruby
# lib/reportportal.rb 
      if config.attributes
        data[:attributes] = config.attributes
      elsif (data[:tags] = config.tags)
```